tiledb/sm/array_schema/array_schema.h

```diff
@@ -360,6 +388,10 @@ class ArraySchema {
 
   /** The array attributes. */
   std::vector<Attribute*> attributes_;
+
+  /** The used name map of the attributes.  */
```

Comment:
Please add some more explanation here (what is the source, what is the destination of the map).

Reply:
I suggest renaming to `attribute_old_to_new_name_map_`

Reply:
changed to attribute_old_to_new_name_map_